app/import_functions.go

```diff
@@ -1229,7 +1238,8 @@ func (a *App) importMultiplePostLines(lines []LineImportWorkerData, dryRun bool)
 		user := users[*line.Post.User]
 
 		// Check if this post already exists.
-		posts, err := a.Srv().Store.Post().GetPostsCreatedAt(channel.Id, *line.Post.CreateAt)
+		var posts []*model.Post
```

Comment:
Not sure why we need to declare this. Are we running into shadowing issues?

Reply:
Fixed